### PR TITLE
feat(oui-tile): create component

### PR DESCRIPTION
**Pull request by whispyy**
UK-5

tile
tile-button
tile-definition
test

### CODE REVIEW

**Comment by whispyy** [comment]
will be rebased.

File: packages/oui-angular/src/index.spec.js [hunk 1]
```diff
@@ -29,8 +29,15 @@ loadTests(require.context("../../oui-chips/src/", true, /.*((\.spec)|(index))$/)
 loadTests(require.context("../../oui-popover/src/", true, /.*((\.spec)|(index))$/));
 loadTests(require.context("../../oui-stepper/src/", true, /.*((\.spec)|(index))$/));
 loadTests(require.context("../../oui-skeleton/src/", true, /.*((\.spec)|(index))$/));
+<<<<<<< HEAD
```
Comment (AxelPeter) [inline packages/oui-angular/src/index.spec.js]: Bad conflict resolution

File: packages/oui-tile/README.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+```html:preview
+<oui-tile data-title="Title">
+    <oui-tile-definition term="Term" description="This is a description"></oui-tile-definition>
+    <oui-tile-definition term="Term">
+        <dd>This is a description</dd>
+    </oui-tile-definition>
+    <oui-tile-definition term="Term" term-popover="This is a popover text" description="This is a description"></oui-tile-definition>
+    <oui-tile-definition term="Progress">
+        <dd>
```
Comment (AxelPeter) [inline packages/oui-tile/README.md]: No need of `<dd>` if it's already added on your template

File: packages/oui-tile/src/definition/tile-definition.component.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    template,
+    controller,
+    bindings: {
+        term: "@?",
+        termPopover: "@?",
+        description: "@?",
+        actions: "=?"
+    },
+    transclude: {
+        descriptionSlot: "?dd",
```
Comment (AxelPeter) [inline packages/oui-tile/src/definition/tile-definition.component.js]: Maybe we should check with something else than `<dd>`.
What do you think about `<oui-tile-description>` ?

File: packages/oui-tile/src/tile.html [hunk 4]
```diff
@@ -0,0 +1,7 @@
+<div class="oui-tile">
+    <h4 class="oui-header_4 oui-tile__title" ng-bind="$ctrl.title"></h4>
```
Comment (AxelPeter) [inline packages/oui-tile/src/tile.html]: one time binding ?

File: packages/oui-tile/src/tile.html [hunk 5]
```diff
@@ -0,0 +1,7 @@
...
+    <p class="oui-tile__legend" ng-if="!!$ctrl.legend" ng-bind="$ctrl.legend"></p>
```
Comment (AxelPeter) [inline packages/oui-tile/src/tile.html]: Same here :)

**Comment by whispyy** [comment]
all corrected

File: packages/oui-tile/src/tile.html [hunk 6]
```diff
@@ -0,0 +1,7 @@
+<div class="oui-tile">
+    <h4 class="oui-header_4 oui-tile__title" ng-bind="::$ctrl.heading"></h4>
+    <p class="oui-tile__legend" ng-if="!!$ctrl.description" ng-bind="::$ctrl.description"></p>
+    <div class="oui-tile__body">
+        <div ng-transclude></div>
```
Comment (AxelPeter) [inline packages/oui-tile/src/tile.html]: (optional) Why not setting the transclude directly on `oui-tile__body` ?

Comment (whispyy) [reply packages/oui-tile/src/tile.html]: Yup u right!

File: packages/oui-tile/README.md [hunk 7]
```diff
@@ -0,0 +1,82 @@
...
+    <oui-tile-definition term="Term" term-popover="This is a popover text" description="This is a description"></oui-tile-definition>
+    <oui-tile-definition term="Progress">
+        <oui-tile-description>
+            <progress class="oui-progress oui-progress_success" value="10" max="100"></progress>
+        </oui-tile-description>
+    </oui-tile-definition>
+    <oui-tile-definition
+        term="Term"
+        description="This is a description">
+        <oui-action-menu data-compact>
```
Comment (cbourgois) [inline packages/oui-tile/README.md]: \+ `data-align="end"`

Comment (AxelPeter) [reply packages/oui-tile/README.md]: -- `data-`, we don't do it in our template.

`<oui-action-menu align="end" compact>`

Comment (whispyy) [reply packages/oui-tile/README.md]: Already tried without data for compact attribute but it hasn't worked.

**Pull request merged**